atomics/T1201/T1201.md

```diff
@@ -90,3 +90,16 @@ Lists the password expiration policy to console on CentOS/RHEL/Ubuntu.
 cat /etc/login.defs
 ```
 <br/>
+<br/>
+
```

Comment:
You don't actually need to update this file. You can remove it from the change.

Reply:
Not sure what this means, I thought this is where the description and such went?

I'll leave it in after adding requested link to RSAT tools.  Let me know if I *must* remove to have change approved.